src/compiler/utils/names.ts

```diff
@@ -117,12 +117,6 @@ export const reserved = new Set([
 	'yield'
 ]);
 
-const void_element_names = /^(?:area|base|br|col|command|embed|hr|img|input|keygen|link|meta|param|source|track|wbr)$/;
-
-export function is_void(name: string) {
-	return void_element_names.test(name) || name.toLowerCase() === '!doctype';
-}
-
```

Comment:
I moved this code to the shared folder which I created.
Because importing this file failed below error.
And because this code is shared code between compiler and runtime.
Is this right way?
And if yes, should I move this file also which is shared between compiler and runtime?
https://github.com/sveltejs/svelte/blob/master/src/compiler/compile/render_ssr/handlers/shared/boolean_attributes.ts

```sh
> agadoo internal/index.mjs

'acorn' is imported by internal/index.mjs, but could not be resolved – treating it as an external dependency
import 'acorn';
```

Reply:
not sure what the error is about, but yea i guess it's okay to move this and `boolean_attribute` over to the `shared` folder

Reply:
I moved `boolean_attribute` also to shared folder.